webapp/app.py

```diff
@@ -37,3 +39,63 @@ def get_tree(uri: str, branch="main"):
 @login_required
 def index(path):
     return render_template("index.html")
+
+
+@app.route('/get-users/<username>', methods=['GET'])
+@login_required
+def get_users(username: str):
```

Comment:
Can we set the capitalisation before sending the query to directory-api? So that it is not case-sensitive when performing a GET request.